shared/src/test/java/uk/gov/di/authentication/shared/entity/AuthenticationAttemptsTest.java

```diff
@@ -14,8 +14,8 @@ class AuthenticationAttemptsTest {
     void testFluentSetters() {
         AuthenticationAttempts attempts = new AuthenticationAttempts();
 
-        AuthenticationAttempts result = attempts.withAttemptIdentifier("attempt-1234");
-        assertEquals("attempt-1234", result.getAttemptIdentifier());
+        AuthenticationAttempts result = attempts.withInternalSubjectId("attempt-1234");
```

Comment:
Minor, should we update this value to look more like a subject id (or just sub out attempt for subject)